src/util.h

```diff
@@ -1,63 +1,84 @@
-void compute_window_squared_sum(size_t n_fft, size_t hop, size_t n_frames, float * tgt, float * window);
+// currently this assumes a center view in which the output vector is
+// reflectively padded by n_fft / 2 on each side.
+void compute_window_squared_sum(size_t n_fft, size_t hop, size_t n_frames,
+                                float *tgt, float *window);
 
-// these functions wrap the stft and istft ggml ops and compute the necessary view and division ops for their indepentent settings.
-struct ggml_tensor * stft(ggml_context * ctx, struct ggml_tensor * a, struct ggml_tensor * window, size_t n_fft, size_t hop, bool abs_and_angle, bool one_sided);
-struct ggml_tensor * istft(ggml_context * ctx, struct ggml_tensor * a, struct ggml_tensor * window_squared_sum, struct ggml_tensor * window, size_t n_fft, size_t hop, bool abs_and_angle, bool one_sided);
+// STFT/ISTFT functions removed - they were using custom GGML operations
+// that don't exist in standard llama.cpp/ggml. If needed, they should be
+// implemented using standard GGML operations or as custom map operations.
 
 // This is a custom op for sine_generation in the Kokoro model.
-void uv_noise_compute(struct ggml_tensor * dst, const struct ggml_tensor * a, const struct ggml_tensor * b, const struct ggml_tensor * c, int ith, int nth, void * userdata);
+void uv_noise_compute(struct ggml_tensor *dst, const struct ggml_tensor *a,
+                      const struct ggml_tensor *b, const struct ggml_tensor *c,
+                      int ith, int nth, void *userdata);
 
 // This is a custom op for logit correction in the Dia model.
-void cfg_scale(struct ggml_tensor * dst, const struct ggml_tensor * a, const struct ggml_tensor * b, int ith, int nth, void * userdata);
+void cfg_scale(struct ggml_tensor *dst, const struct ggml_tensor *a,
+               const struct ggml_tensor *b, int ith, int nth, void *userdata);
 
 bool has_suffix(std::string value, std::string suffix);
 bool has_prefix(std::string value, std::string prefix);
 
-std::vector<std::string> split(std::string target, std::string split_on, bool include_split_characters = false);
-std::vector<std::string> split(std::string target, const char split_on, bool include_split_characters = false);
+std::vector<std::string> split(std::string target, std::string split_on,
+                               bool include_split_characters = false);
+std::vector<std::string> split(std::string target, const char split_on,
+                               bool include_split_characters = false);
 std::string strip(std::string target, std::string vals = " ");
-std::string replace_any(std::string target, std::string to_replace, std::string replacement);
+std::string replace_any(std::string target, std::string to_replace,
+                        std::string replacement);
 
-[[noreturn]] void tts_abort(const char * file, int line, const char * fmt, ...);
+[[noreturn]] void tts_abort(const char *file, int line, const char *fmt, ...);
```

Comment:
Why was this spacing changed away from the ggml-org code style?

Reply:
@danielzgtg, that's a good question. Wasn't intentional.  I'll see if I can revert that.